### ISSUE TITLE
Stacks: Lightroom sequencial naming

### ISSUE THREAD

**Issue by Rolf-Smit** [photoprism/photoprism#2241]
**Problem:**

When exporting edited JPEG files from Adobe Lightroom Classic, Lightroom will name the newly generated JPEG files by adding a `-x` postfix. For example DSC00128.JPG becomes DSC00128-2.JPG.

Currently these photo's end up as separate photos in PhotoPrism, and PhotoPrism is unable to stack them.

**Expected behavior:**

Currently it seems only the ` (x)` prefix and ` copy x` postfixes are considered "sequential files". However since Adobe Lightroom Classic is a very popular photo tool, it could make sense to also include the `-x` postfix?

In other words it would be nice if PhotoPrism would also stack photo's that have the same name and folder ignoring the `-x` as postfix.

_Note: reported as a bug, but maybe it is a feature request after all?_

**Comment by graciousgrey** [comment]
Could you give us some more example filenames? I don't see the -x in your example or is x a placeholder for a number?

**Comment by Rolf-Smit** [comment]
Yes `x` is meant a s a placeholder for a number. As far as I can tell Lightroom works like this:

Original file name:  `DSC00128.JPG`

If Lightroom needs to create a file with exactly the same path, name and extension, the next file name will be: `DSC00128-2.JPG`, a third file will become `DSC00128-3.JPG`.

This for example happens when you edit a JPEG file in Lightroom, then export the edited file as JPEG to the original location (with override disabled).


_Happy to contribute this improvement myself, if accepted._

**Comment by Rolf-Smit** [comment]
@graciousgrey is this an acceptable feature? Please let me know, I would not mind working on this.

**Comment by graciousgrey** [comment]
In general yes, but we should do some research first. 

Because the pattern -x is so simple there is a high risk that many files get stacked that should not be stacked. This would cause lot's of manual work for people to unstack them again. So it is most probably not a good idea to just add the pattern and stack files if "sequential names" is enabled.

We might need an additional config option for this in addition to the general "sequential name" option. Or we only stack files with this pattern if in addition the metadata contains information that he file was created by Lightroom.

Also it would be interesting to know if Lightroom always uses this pattern or if there is a choice for the user and if so what is the default pattern.

Any ideas or input regarding this is much appreciated. 

You find more information on acceptance criteria for pull requests here: https://docs.photoprism.app/developer-guide/pull-requests/

Somehow related but easier: https://github.com/photoprism/photoprism/issues/1122
